ops/README.md

```diff
@@ -0,0 +1,61 @@
+# Operations
+========================
+
+This directory contains files related to deploying or running the application(s).
+
+## Authenticating
+
+You'll need the [Cloud Foundry CLI](https://docs.cloud.gov/getting-started/setup/).
+
+We use the V7 Cloud Foundry CLI.
+
+```shell
+cf login -a api.fr.cloud.gov --sso
+```
+
+After authenticating, make sure you are targeting the correct org and space!
+
+```bash
+cf spaces
+cf target -o <ORG> -s <SPACE>
+```
+
+## Rotating Secrets
```

Comment:
Can we specify "environment secrets" here?

Reply:
Done.